src/helpers/helpers.dom.js

```diff
@@ -56,7 +57,7 @@ function getConstraintDimension(domNode, maxStyle, percentageProperty) {
 			hasCContainer ? parseMaxStyle(constrainedContainer, parentNode, percentageProperty) : infinity);
 	}
 
-	return 'none';
+	return undefined;
```

Comment:
When the function returns undefined it looks like an error. Why not return null for example?